lambdatimes/src/components/Content/Tab.js

```diff
@@ -7,7 +7,7 @@ const Tab = props => {
   return (
     <div
       className={''}
```

Comment:
here you will want to do a ternary that asks if the props.tab matches the selected tab...if it does the class name will have active in it. If it doesn't, it will just be tab